src/components/DashboardPage/AccountCard.js

```diff
@@ -57,35 +57,36 @@ const AccountCard = ({
         />
       </Box>
     </Box>
-    <CalculateBalance
-      counterValue={counterValue}
-      accounts={[account]}
-      daysCount={daysCount}
-      render={({ allBalances, totalBalance, refBalance }) => (
+    <CalculateBalance counterValue={counterValue} accounts={[account]} daysCount={daysCount}>
+      {({ isAvailable, balanceHistory, balanceStart, balanceEnd }) => (
         <Box flow={4}>
           <Box flow={2} horizontal>
             <Box justifyContent="center">
-              <FormattedVal
-                animateTicker
-                fiat={counterValue}
-                val={totalBalance}
-                alwaysShowSign={false}
-                showCode
-                fontSize={3}
-                color="graphite"
-              />
+              {isAvailable ? (
+                <FormattedVal
+                  animateTicker
+                  fiat={counterValue}
+                  val={balanceEnd}
+                  alwaysShowSign={false}
+                  showCode
+                  fontSize={3}
+                  color="graphite"
+                />
+              ) : null}
             </Box>
             <Box grow justifyContent="center">
-              <FormattedVal
-                isPercent
-                val={Math.floor((totalBalance - refBalance) / refBalance * 100)}
-                alwaysShowSign
-                fontSize={3}
-              />
+              {balanceStart && isAvailable ? (
+                <FormattedVal
+                  isPercent
+                  val={Math.floor((balanceEnd - balanceStart) / balanceStart * 100)}
```

Comment:
we need a `DeltaChange` component to factorize the logic (because we actually want to avoid `/ 0` & also if % is too big diverge the rendering)